docker/web_nginx.conf

```
@@ -0,0 +1,100 @@
+
+## Gio and Sam
+# added an haproxy to better balance the requests
+# between the webnodes
+# http://www.openlibrary.org/admin?stats
+#
+#
+# old conf (requests balanced by nginx)
+#upstream webnodes {
+#    server ol-web1.us.archive.org:8080;
+#    server ol-web2.us.archive.org:8080;
+#}
+#
+# using haproxy
+upstream webnodes {
+    server web_haproxy:7072;
+}
+
+server {
+    listen       80;
+    listen      443;
+    server_name  openlibrary.org;
+
+    include /run/secrets/petabox_seed;
```

Lines like this also need to be removed.
```suggestion
    include /run/secrets/petabox_seed;
```

---

Fixed